about.html

```diff
@@ -0,0 +1,110 @@
+<!DOCTYPE html>
+<html>
+<head>
+	<title>Who is Ralph?</title>
+	<link rel="stylesheet" type="text/css" href="css/global.css">
+	<link rel="stylesheet" type="text/css" href="css/grid.css">
+	<link rel="stylesheet" type="text/css" href="css/nav.css">
+	<link rel="stylesheet" type="text/css" href="css/top-bar.css">
+	<link rel="stylesheet" type="text/css" href="css/content.css">
+	<link rel="stylesheet" type="text/css" href="css/header.css">
+	<link rel="stylesheet" type="text/css" href="css/typography.css">
+	<link rel="stylesheet" type="text/css" href="css/footer.css">
+</head>
+<body>
+	<div id="top-bar" class="col4">
+		<img src="images/basketball-1511298_1920.jpg" alt="basketball player">
+		<img src="images/mathematics-1509559_1920.jpg" alt="maths on a blackboard">
+		<img src="images/juggler-1216853_1920.jpg" alt="juggler">
+		<img src="images/crowd-1056764_1920.jpg" alt="crowd in a nightclub">
+	</div>
+	<header>
+		<a class="nav-button previous" href="blog.php?page=most-recent">
+			<img src="images/blog-icon.png" alt="blog icon">
+		</a>
+		<img class="centre" src="images/ralphportrait.jpg" alt="portrait of ralph">
+		<h1 class="small-heading">Ralph Rutter</h1>
+		<a class="nav-button next" href="index.html">
+			<img src="images/home-icon.png" alt="home icon">
+		</a>
+	</header>
+	<main>
+		<div class="container margin">
+			<div class="annex pre">
+				<img src="images/structure-1473346_1920.jpg alt="fractal"">
```

Comment:
Amended my incorrect amendment, in commit b6636e7a7136e6ded2f0b58baa296c29f9f9ec1c